src/databaseaccessor/commentdb.h

```diff
@@ -0,0 +1,25 @@
+#ifndef COMMENTDB_H
+#define COMMENTDB_H
+#include "accessor.h"
+#include "structsfordb.h"
+
+class CommentDb : public Accessor
+{
+public:
+    CommentDb();
+    void addCommentsToDb(const QVector<Comment> &comments);
+    void deleteCommentFromDb(const int commentLine, const QString commentFile);
+    void deleteCommentsFromDb(const QString commentFile);
```

Comment:
In most cases methods of Accessor descendants are passed QString by value
Is there any necessity to copy arguments?